README.md

```diff
@@ -0,0 +1,8 @@
+[![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-pavlenk0.png)](https://travis-ci.org/kpi-web-guild/django-girls-blog-pavlenk0)
```

Comment:
normally it's better to specify exact branch for the badge. you don't want to a have red badge on the glance of your repo just because the last build was ran for some unfinished feature on another branch for which it's okay to fail. also using vector version of graphics is preferable on the web:
```
[![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-pavlenk0.svg?branch=master)](https://travis-ci.org/kpi-web-guild/django-girls-blog-pavlenk0)
```